tools/Packer/ConfigModel.cs

```diff
@@ -30,5 +30,10 @@ public static ConfigModel FromFile(string path)
 
             return JsonConvert.DeserializeObject<ConfigModel>(File.ReadAllText(path), settings);
         }
+
+        public void ToFile(string path)
```

Comment:
Nit: Rename `WriteToFile`

`ToFoo` is a convention that would usually have the following signature: `Foo ToFoo()` such that it does a conversion